examples/README.md

```diff
@@ -0,0 +1,30 @@
+# Gorums examples
+
+This folder contains examples of services implemented with Gorums.
+If you want a detailed walkthrough of how to get started with Gorums, read the [user guide](../doc/userguide.md).
+
+## Prerequisites
+
+Requires Go 1.13 or later and you must have `$GOPATH/bin` in your `$PATH`.
```

Comment:
I think it's still relevant to use `GOPATH` here though. That way you can simply run `go get github.com/relab/gorums/examples/storage`. And then the `storage` binary will be available. I've always thought of `GOPATH/bin` as the "default", and `GOBIN` as an override.

Reply:
ok. Maybe we can just link to the official [instructions](https://golang.org/doc/code.html#Command), to avoid being specific about `GOPATH`. They also use this:

```
$ go get golang.org/x/tour
This will place the tour binary in your workspace's bin directory.
```

Reply:
Looks like they mean GOPATH when they say "workspace"
https://web.archive.org/web/20190806061015/https://golang.org/doc/code.html#Workspaces

Reply:
*"the GOPATH environment variable specifies the location of your workspace"*